425 docs rfcadd docs for the mcp server (#4477)
timescale/docs: _partials/_devops-mcp-commands.md

> You can use the following commands with $MCP_LONG.

Reviewer: ```suggestion
You can use the following commands with $MCP_LONG:
```

> | Command                         | Parameter         | Required | Description                                                                                                                                                                                                                                                                                                     |

Reviewer: This whole table would benefit from variables for 'service', some rephrasing for readability and clarity, definite articles, and code font for examples and defaults.

Author: Done

|                                 | `with_password`   | -        | Whether to include the password in the response and connection string. **NEVER set to true unless user explicitly requests password.** Default: false                                                                                                                                                           |
| `tiger_service_create`          | -                 | -        | Create a new database service in Tiger with specified configuration. **WARNING: Creates billable resources.** Default behavior returns immediately while service provisions in background (recommended). Setting wait=true blocks for minutes until ready - only use if user explicitly needs immediate access. |
|                                 | `name`            | -        | Human-readable name for the service (auto-generated if not provided). Max 128 characters. Examples: "my-production-db", "analytics-service", "user-store"                                                                                                                                                       |

> |                                 | `addons`          | -        | Array of addons to enable for the service. Options: <ul><li>time-series (enables TimescaleDB)</li><li>ai (enables AI/vector extensions)</li></ul> Use empty array for PostgreSQL-only.                                                                                                                          |

Reviewer: ```suggestion
|                                 | `addons`          | -        | Array of addons to enable for the service. Options: <ul><li>time-series (enables TimescaleDB)</li><li>ai (enables AI/vector extensions)</li></ul> Use empty array for Postgres-only.                                                                                                                          |
```

> You can use the following commands with $MCP_LONG:

Reviewer 2: These are MCP "tools", not "commands". And _you_ don't really use them - your AI assistant does. I would say something more like:

```suggestion
$MCP_LONG exposes the following MCP tools to your AI assistant:
```

> $MCP_LONG exposes the following MCP tools to your AI assistant:

Reviewer: ```suggestion
$MCP_LONG exposes the following MCP tools to your AI Assistant:
```

|                          | `service_id`        | ✓        | The unique identifier of the $SERVICE_SHORT (10-character alphanumeric string).                                                                                                                                                                                                          |
|                          | `with_password`     | -        | Set to `true` to include the password in the response and connection string. <br/> **WARNING**: never do this unless the user explicitly requests the password.                                                                                                                          |
| `service_create`         | -                   | -        | Create a new $SERVICE_SHORT in $CLOUD_LONG. <br/> **WARNING**: creates billable resources.                                                                                                                                                                                               |

> |                          | `name`              | -        | Set the human-readable name of up to 128 character for this $SERVICE_SHORT.                                                                                                                                                                                                              |

Reviewer: ```suggestion
|                          | `name`              | -        | Set the human-readable name of up to 128 characters for this $SERVICE_SHORT.                                                                                                                                                                                                              |
```

| `service_update_password` | -                   | -        | Update the password for the `tsdbadmin` for this $SERVICE_SHORT. The password change takes effect immediately and may terminate existing connections.                                                                                                                                    |
|                          | `service_id` | ✓        | The unique identifier of the $SERVICE_SHORT you want to update the password for.                                                                                                                                                                                                         |
|                          | `password`          | ✓        | The new password for the `tsdbadmin` user.                                                                                                                                                                                                                                               |

> | `db_execute_query`                       | -                   | -        | Execute a single SQL query against a $SERVICE_SHORT. This command returns column metadata, result rows, affected row count, and execution time. Multi-statement queries not supported.  <br/> **WARNING**: can execute destructive SQL including INSERT, UPDATE, DELETE, and DDL commands. |

Reviewer: ```suggestion
| `db_execute_query`                       | -                   | -        | Execute a single SQL query against a $SERVICE_SHORT. This command returns column metadata, result rows, affected row count, and execution time. Multi-statement queries are not supported.  <br/> **WARNING**: can execute destructive SQL including INSERT, UPDATE, DELETE, and DDL commands. |
```

|                          | `service_id` | ✓        | The unique identifier of the $SERVICE_SHORT. Use `tiger_service_list` to find $SERVICE_SHORT IDs.                                                                                                                                                                                        |
|                          | `query`             | ✓        | The SQL query to execute. Single statement queries are supported.                                                                                                                                                                                                                        |
|                          | `parameters`        | -        | Query parameters for parameterized queries. Values are substituted for the `$n` placeholders in the query.                                                                                                                                                                               |

> |                          | `timeout_seconds`   | -        | The query timeout in seconds. Default: 30.                                                                                                                                                                                                                                               |

Reviewer: ```suggestion
|                          | `timeout_seconds`   | -        | The query timeout in seconds. Default: `30`.                                                                                                                                                                                                                                               |
```

|                          | `role`              | -        | The $SERVICE_SHORT role/username to connect as. Default: `tsdbadmin`.                                                                                                                                                                                                                    |

> |                          | `pooled`            | -        | Use [connection pooling][Connection pooling]. This is only available if you have already enabled it for the $SERVICE_SHORT. Default: `false`                                                                                                                                             |

Reviewer: ```suggestion
|                          | `pooled`            | -        | Use [connection pooling][Connection pooling]. This is only available if you have already enabled it for the $SERVICE_SHORT. Default: `false`.                                                                                                                                             |
```
